cli/cmd/install.go

```diff
@@ -233,10 +240,32 @@ func doInstallation() error {
 	}
 
 	logging.PrintLog("Keptn has been successfully set up on your cluster.", logging.InfoLevel)
+
+	// Hide sensitive information like api-token and endpoint in post-installation output
+	if *installParams.HideSensitiveData {
+		return nil
+	}
+
 	logging.PrintLog("---------------------------------------------------", logging.InfoLevel)
-	fmt.Println("* To quickly access Keptn, you can use a port-forward and then authenticate your Keptn CLI (in a Linux shell):\n" +
-		" - kubectl -n " + keptnNamespace + " port-forward service/api-gateway-nginx 8080:80\n" +
-		" - keptn auth --endpoint=http://localhost:8080/api --api-token=$(kubectl get secret keptn-api-token -n " + keptnNamespace + " -ojsonpath={.data.keptn-api-token} | base64 --decode)\n")
+
+	apiToken, err := getAPITokenFromSecret(keptnNamespace)
+	if err != nil {
+		apiToken = "$(kubectl get secret keptn-api-token -n " + keptnNamespace + " -ojsonpath={.data.keptn-api-token} | base64 --decode)"
+	}
+
+	if installParams.EndPointServiceType.String() == "NodePort" || installParams.EndPointServiceType.String() == "LoadBalancer" {
+		endpoint, err := getAPIEndpoint(keptnNamespace, installParams.EndPointServiceType.String())
+		if err == nil {
+			showFallbackConnectMessage = false
+			fmt.Println("* To quickly access Keptn, you can authenticate your Keptn CLI (in a Linux shell):\n" +
+				" - keptn auth --endpoint=" + endpoint + " --api-token=" + apiToken + "\n")
```

Comment:
In a previous commit, you used the command: 
`$(kubectl get secret keptn-api-token -n " + keptnNamespace + " -ojsonpath={.data.keptn-api-token} | base64 --decode)` instead of showing the `api-token`.

I really liked it since then we minimize the risk of leaking the API token. Can you please revert it to use: `$(kubectl get secret keptn-api-token -n " + keptnNamespace + " -ojsonpath={.data.keptn-api-token} | base64 --decode)`

Reply:
Sure, will make the changes, but then it won't be visible in any cases.

Reply:
But that is okay. 
We actually don't want to show the API token, but rather help the user to run the `keptn auth` command.

Reply:
Done